tests/serialization_tests/Cargo.toml

```diff
@@ -10,7 +10,7 @@ serde_tests = ["serde", "crypto", "base64"]
 [dependencies]
 serde = { version = "1.0", features = ["derive"], optional = true }
 crypto = { path = "../../crypto", optional = true }
-base64 = { version = "0.12.1", optional = true }
+base64 = { version = "0.12.0", optional = true }
```

Comment:
out of curiosity, why downgrade here and upgrade in `vm/runtime/Cargo.toml`? It all gets resolved the same but curious

Reply:
Oops i missed that.